src/test/java/com/wikia/webdriver/common/dataprovider/RandomDataProvider.java

```diff
@@ -0,0 +1,16 @@
+package com.wikia.webdriver.common.dataprovider;
+
+import java.util.Date;
+
+public class RandomDataProvider {
```

Comment:
@ludwikkazmierczak We can create ticket for replacing all getTimestamp() usages with that DateTime.now().getMillis() substitute. Should I do that ?